e2e/testdata/fn-eval/include-meta-resources-v1alpha1/.expected/config.yaml

```diff
@@ -16,5 +16,6 @@ runCount: 2
 testType: eval
 image: gcr.io/kpt-fn/set-namespace:v0.1.3
 includeMetaResources: true
+exitCode: 1
```

Comment:
Why?

Reply:
Actually this PR removes `--includeMetaResources` commandline flag, so instead of deleting this test, I re-purposed to test the error if someone specifies this flag. I should probably add comment explaining that.